roles/rsyslog/templates/etc/rsyslog.conf.j2

```diff
@@ -1,49 +1,56 @@
 # {{ ansible_managed }}
 
-{% if not rsyslog_default|default(true)|bool %}
+{% if rsyslog_use_roles|default(false)|bool %}
```

Comment:
So this is the default rhel7 rsyslog config file, just like the one provided by the rsyslog rpm package?  How different is it from the default on rhel8?

Reply:
Thanks, @richm .  Per discussion on IRC, eliminating rsyslog_use_roles and changing the behavior to NOOP if no LSR/Logging variables are given to the inventory file.